libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -151,12 +153,23 @@ public void cancelFeedback() {
   /**
    * Returns the current instance of {@link MapboxNavigation}.
    *
-   * @since 0.6.1
+   * Will be null if navigation has not been initialized.
    */
-  public MapboxNavigation getNavigation() {
+  @Nullable
+  public MapboxNavigation retrieveNavigation() {
```

Comment:
Yes I think it makes sense to align the "getter" naming for `NavigationView`

Reply:
Yeah, I agree.
Side note - I like to add this kind of comments so later is easier to track what PRs / changes need to be highlighted in the release notes because introduced breaking changes 🚀